docs/content/latest/releases/whats-new/_index.md

```diff
@@ -0,0 +1,95 @@
+---
+title: What's new in 2.1.5
+headerTitle: What's new in 2.1.5
+linkTitle: What's new in 2.1.5
+description: Enhancements, changes, and resolved issues in the latest YugabyteDB release.
+headcontent: Features, enhancements, and resolved issues in the latest release.
+image: /images/section_icons/quick_start/install.png
+section: RELEASES
+menu:
+  latest:
+    identifier: whats-new
+    weight: 2589 
+---
+
+**Released:** April 27, 2020.
+
+**New to YugabyteDB?** Follow [Quick start](../../quick-start/) to get started and running in less than five minutes.
+
+## Downloads
+
+### Binaries
+
+<a class="download-binary-link" href="https://downloads.yugabyte.com/yugabyte-2.1.5.0-darwin.tar.gz">
+  <button>
+    <i class="fab fa-apple"></i><span class="download-text">macOS</span>
+  </button>
+</a>
+&nbsp; &nbsp; &nbsp; 
+<a class="download-binary-link" href="https://downloads.yugabyte.com/yugabyte-2.1.5.0-linux.tar.gz">
+  <button>
+    <i class="fab fa-linux"></i><span class="download-text">Linux</span>
+  </button>
+</a>
+<br />
+
+### Docker
+
+```sh
+docker pull yugabytedb/yugabyte:2.1.5.0-b14
+```
+
+## YSQL changes
```

Comment:
drop `changes` and keep it simply as `YSQL`

Reply:
fixed